packages/editor/src/browser/navigation/navigation-location-service.ts

```diff
@@ -165,12 +169,13 @@ export class NavigationLocationService {
     }
 
     /**
-     * Clears the navigation history.
+     * Clears the total history.
```

Comment:
+ The idea is to avoid having multiple stacks in different places because it's hard to maintain in that case. For example, in this method, we can call 1 `clear()` from the service instead of calling 2 `clear` functions from 2 different services.
+ It's also to align with vscode (these 2 stacks in one single service).

Reply:
Sounds good to me then, thanks for the explanation.